Source/FieldSolver/ImplicitSolvers/ImplicitSolver.cpp

```diff
@@ -69,6 +69,8 @@ Array<LinOpBCType,AMREX_SPACEDIM> ImplicitSolver::convertFieldBCToLinOpBC (const
         } else if (a_fbc[i] == FieldBoundaryType::Neumann) {
             // Also for FieldBoundaryType::PMC
             lbc[i] = LinOpBCType::Neumann;
+        } else if (a_fbc[i] == FieldBoundaryType::PECInsulator) {
+            lbc[i] = LinOpBCType::Dirichlet;
```

Comment:
Maybe we could raise a warning here, since this is a temporary fix. 
Something like:
```
ablastr::warn_manager::WMRecordWarning(
    "Implicit solver",
    YOUR_WARNING_MESSAGE,
    ablastr::warn_manager::WarnPriority::medium);
```
What do you think?